backend/src/utils/db_utils.py

```
@@ -2012,4 +2025,183 @@ def cleanup_expired_export_jobs() -> int:
+def create_import_job(import_job: 'ImportJob') -> None:
+    """
+    Create a new import job in DynamoDB.
+    
+    Args:
+        import_job: The ImportJob object to store
+        
+    Raises:
+        ClientError: If there's a DynamoDB error
+    """
+    try:
+        from models.import_job import ImportJob
```

Ruff (0.12.2)

`models.import_job.ImportJob` imported but unused

Remove unused import: `models.import_job.ImportJob`

(F401)